src/Settings.php

```diff
@@ -126,6 +132,14 @@ public static function parseArguments(array $arguments)
     {
         $arguments = new ArrayIterator(array_slice($arguments, 1));
         $settings = new self;
+        
+        $git = "git show HEAD \
+           --name-only \
+           --diff-filter=ACM \
+           -m \
+           --first-parent \
+           --format=format: \
+           -- "${PATH_ARGS[@]}" | egrep -v '^$' || :";
```

Comment:
What about move this lines to method? Something like `createCommandForChangedFiles`?